addons/knowledge/static/src/js/field_html.js

```diff
@@ -0,0 +1,70 @@
+/** @odoo-module */
+
+import FieldHtml from 'web_editor.field.html';
+import {FieldHtmlInjector} from './knowledge_field_html_injector';
+import {KnowledgePlugin} from './KnowledgePlugin';
+
+FieldHtml.include({
+    /**
+     * @override
+     */
+    _renderReadonly: function () {
+        const prom = this._super.apply(this, arguments);
+        if (prom) {
+            return prom.then(function () {
+                return this._addFieldHtmlInjector();
+            }.bind(this));
+        } else {
+            return this._addFieldHtmlInjector();
+        }
+    },
+    /**
+     * Add a @see FieldHtmlInjector to this field_html, which will add temporary
+     * @see KnowledgeBehavior and/or @see KnowledgeToolbar to @see OdooEditor
+     * blocks. Delegate control of the editor history for those behaviors.
+     * To avoid function duplicates for edit and readonly modes, create history
+     * control dummies.
```

Comment:
One man said : "ou sinon je peux me trimballer avec une reference a l'editeur tout le long"
One other could say : "let's do that"
Personally, I don't know :smile: